config/jobs/kubernetes-csi/csi-driver-host-path/csi-driver-host-path-config.yaml

```diff
@@ -188,7 +188,7 @@ presubmits:
           cpu: 2000m
   - name: pull-kubernetes-csi-csi-driver-host-path-1-20-on-kubernetes-1-20
     always_run: true
-    optional: true
+    optional: false
```

Comment:
Why is this changed to "false" by default now?

Reply:
Because we no longer want to tolerate test failures on 1.20.